documentation/modules/proc-applying-infra-config.adoc

```diff
@@ -20,15 +17,15 @@ ifeval::["{cmdcli}" == "oc"]
 {cmdcli} login -u developer
 ----
 
-. Select the project where {ProductName} is installed:
+. Change to the project where {ProductName} is installed:
 +
 [subs="attributes",options="nowrap"]
 ----
-{cmdcli} project enmasse
+{cmdcli} project _{ProductNamespace}_
 ----
 endif::[]
 
-. Create infrastructure configuration:
+. For a new infrastructure configuration, create the infrastructure configuration:
```

Comment:
Sounds a bit redundant. Maybe just, "Create the infrastructure configuration:"

Reply:
@bhardesty, Since this procedure is for both creating a new config and editing an existing one, I added that verbiage, but obviously forgot to add something about "or edit an existing config". Will fix that.

Reply:
Looks good.